packages/babel-parser/src/types.js

```diff
@@ -849,6 +850,20 @@ export type ClassPrivateProperty = NodeBase & {
   override?: true,
 };
 
+export type ClassAccessorProperty = ClassMemberBase &
+  DeclarationBase & {
+    type: "ClassAccessorProperty",
+    key: Expression | PrivateName,
+    value: ?Expression, // TODO: Not in spec that this is nullable.
```

Comment:
Nit: This todo item can be removed. It is extremely out-of-dated as there is `PropertyDefinition` in estree with nullable `value`: https://github.com/estree/estree/blob/master/es2022.md#propertydefinition